reference/modules/authorization.md

```diff
@@ -65,17 +65,56 @@ No relations are supported for this type
+| **View**       | `view`       | Allows viewing tasks.                                                    |
+| **View list**  | `view_list`  | Allows retrieving lists of tasks.                                        |
+| **Claim**      | `claim`      | Allows users to assign themselves as the task                            |
+| **Assign**     | `assign`     | Allows assigning a tasks handler of both the current and other users.    |
+| **Assignable** | `assignable` | Allows users with permissions on this action to be assigned to the task. |
+| **Complete**   | `complete`   | Allows users to complete the task.                                       |
 
 #### Supported relations
 
 | Related resource |
 |------------------|
 | **Document**     |
+
+## Supported conditions
+The following conditions can be used with their respective structures:
+
+| Resource name                 | Key          | Description                                                                                          |
+|-------------------------------|:-------------|:-----------------------------------------------------------------------------------------------------|
+| **[Field](#field)**           | `field`      | Allows conditions to be applied to specific fields.                                                  |
+| **[Expression](#expression)** | `expression` | Allows JsonPath expressions to search through JSON.                                                  |
+| **[Container](#container)**   | `container`  | Allows more complex conditions by denoting the related entity the nested conditions should apply to. |
+
+### Field
+
+| key        | Description                                     | Examples                                  |
+|:-----------|:------------------------------------------------|:------------------------------------------|
+| `type`     | The type of condition. In this case `field`.    | `field`                                   |
+| `field`    | The field the value should be compared against. | `assigneeId`, `documentDefinitionId.name` |
+| `operator` | The operator for the comparison.                | `==`, `<`                                 |
+| `value`    | The value to compare the field against.         | `20000`, `${currentUserId}`               |
+
+
+### Expression
+
+| key        | Description                                       | Examples                    |
+|:-----------|:--------------------------------------------------|:----------------------------|
+| `type`     | The type of condition. In this case `expression`. | `expression`                |
+| `field`    | The field used to start the path from.            | `content.content`           |
+| `path`     | The path that points to the field to compare to.  | `$.height`                  |
+| `operator` | The operator for the comparison.                  | `==`, `<`                   |
```

Comment:
Do we have the possible operators documented as well?

Reply:
Now we do